src/devices/Adg731/samples/Program.cs

```diff
@@ -0,0 +1,67 @@
+﻿using System;
+using System.Device.Spi;
+using System.Diagnostics;
+using System.Threading;
+using Iot.Device.Adg731;
+
+namespace ADG731Sample
+{
+    internal class Program
+    {
+        private static Adg731 s_adg731;
+
+        public static void Main()
+        {
+            WaitForDebugger();
+
+            // Digilent Pmod HAT Adapter
+            // JA - Bus 0, CS 0
+            // JB - Bus 0, CS 1
+            SpiConnectionSettings settings = new SpiConnectionSettings(0, 1)
+            {
+                ClockFrequency = ((ISpiDeviceMetadata)Adg731.GetDeviceMetadata()).MaximumSpiFrequency,
+                Mode = SpiMode.Mode3
+            };
+            SpiDevice adg731SpiDevice = SpiDevice.Create(settings);
+
+            Console.WriteLine($"Connecting to ADG731 using SPI {adg731SpiDevice.ConnectionSettings.Mode.ToString()} at {adg731SpiDevice.ConnectionSettings.ClockFrequency / 1000.0:N1} kHz...");
+
+            s_adg731 = new Adg731(adg731SpiDevice);
+            s_adg731.IsEnabled = true;
+            s_adg731.IsSelected = true;
+
+            int loopcounter = 0;
+            while (true)
+            {
+                s_adg731.ActiveChannel = loopcounter % ((IMultiplexerDeviceMetadata)Adg731.GetDeviceMetadata()).MultiplexerChannelCount;
+                Console.WriteLine($"Channel changed to {s_adg731.ActiveChannel}");
+                loopcounter++;
+                Thread.Sleep(500);
+            }
+        }
+
+        private static void WaitForDebugger()
```

Comment:
shouldn't be part of the sample (btw you can search for Scott Hanselman post about debugging on Pi - link should be somewhere here in the repo)